crates/ruff/src/rules/flake8_tidy_imports/relative_imports.rs

```diff
@@ -106,25 +106,25 @@ fn fix_banned_relative_import(
 
         let module_name = if let Some(module) = module {
             let call_path = from_relative_import(&parts, module);
-            // Require import to be a valid PEP 8 module:
+            // Require import to be a valid module:
             // https://python.org/dev/peps/pep-0008/#package-and-module-names
-            if !call_path.iter().all(|part| is_module_name(part)) {
+            if !call_path.iter().all(|part| is_identifier(part)) {
```

Comment:
Are module names allowed to use keywords? Is that defined in the spec anywhere?

Reply:
I asked ChatGpt :sweat_smile: 
![img](https://live.staticflickr.com/65535/52779854898_ee9913da87_c.jpg)